pkg/kubectl/cmd/portforward_test.go

```diff
@@ -47,23 +47,25 @@ func testPortForward(t *testing.T, flags map[string]string, args []string) {
 	version := "v1"
 
 	tests := []struct {
-		name            string
-		podPath, pfPath string
-		pod             *api.Pod
-		pfErr           bool
+		name                          string
+		podPath, fetchPodPath, pfPath string
+		pod                           *api.Pod
+		pfErr                         bool
 	}{
 		{
-			name:    "pod portforward",
-			podPath: "/api/" + version + "/namespaces/test/pods/foo",
-			pfPath:  "/api/" + version + "/namespaces/test/pods/foo/portforward",
-			pod:     execPod(),
+			name:         "pod portforward",
+			podPath:      "/api/" + version + "/namespaces/test/pods/foo",
+			fetchPodPath: "/namespaces/test/pods/foo",
```

Comment:
this looks like a mistake in how the test client was constructed... the `fetchPodPatch` you added is just `podPath` with the api prefix missing... we shouldn't add both of those

Reply:
I only spent a few min looking for a better solution than this (which is also used in https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/attach_test.go#L190).   I think pkg/kubectl/cmd/util/factory_object_mapping.go is the one making the call without /api/version, but don't know enough yet to resolve that correctly.

Reply:
this fixes it:

```go
tf.Client = &fake.RESTClient{
  VersionedAPIPath:     "/api/v1",
  ...
```

Reply:
That fixed it.  I will file an issue on that so other similar cases can be fixed too.